config/clients/go/template/example/example1/go.mod.mustache

```diff
@@ -7,4 +7,4 @@ require {{gitHost}}/{{gitUserId}}/{{gitRepoId}} v{{packageVersion}}
 require golang.org/x/sync v0.7.0 // indirect
 
 // To reference local build, uncomment below and run `go mod tidy`
-replace {{gitHost}}/{{gitUserId}}/{{gitRepoId}} v{{packageVersion}} => ../../
\ No newline at end of file
+// replace {{gitHost}}/{{gitUserId}}/{{gitRepoId}} v{{packageVersion}} => ../../
```

Comment:
Also for a later PR - I think for all examples, we can make local the default

Reply:
What dya think about eventually getting rid of things like `{{gitHost}}/{{gitUserId}}/{{gitRepoId}}` and things like `{{appshortname}}` in future too now that we have a single SDK publish target? 🥺